src/mapOfPeople.js

```diff
@@ -40,7 +40,17 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const uniquePeople = people.reduce((arrayOfPeople, person, currentIndex) => {
+    const positionOfPerson = element => arrayOfPeople.has(element);
```

Comment:
1. `positionOfPerson ` - is it not function name(`getPersonPosition`)
2. I wrote about it earlier. You can save a result here
```suggestion
    const containsPerson = arrayOfPeople.has(person);
```

and see two next suggestions

Reply:
`has ` is long operation - will be better to perform it once if possible